packages/did/README.md

```diff
@@ -0,0 +1,85 @@
+# @tangleid/did
+
+Utilities for register and resolve Decentralized Identifiers(DIDs).
```

Comment:
Fix the grammar. Should be "Utilities for Decentralized Identifiers(DIDs) registration and resolve".